gengokit/gengokit_test.go

```diff
@@ -172,7 +168,7 @@ func TestApplyTemplateFromPath(t *testing.T) {
 		t.Fatal(err)
 	}
 
-	te, err := newTemplateExecutor(dt, goImportPath)
+	te, err := newTemplateExecutor(dt, goImportPath, goImportPath)
```

Comment:
This change hampers readability. I see three calls to `newTemplateExecutor`, all with the second parameter now duplicated to being the third parameter. As a reader, this raises questions in my mind that ought not to occur, like, "Why do I have to pass the same parameter twice?"

Will these parameters always be the same? If not, why are they the same here?